src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -547,6 +526,48 @@ private bool CheckParameterValueKind(SyntaxNode node, BoundParameter parameter,
             return true;
         }
 
+        private static bool CheckParameterRefEscape(SyntaxNode node, BoundParameter parameter, uint escapeTo, bool checkingReceiver, DiagnosticBag diagnostics)
+        {
+            ParameterSymbol parameterSymbol = parameter.ParameterSymbol;
+            var paramKind = parameterSymbol.RefKind;
+
+            // byval parameters can escape to method's top level.
```

Comment:
Does "method's top level scope" refer to the local function definition for parameters of a local function?

Reply:
nearest containing method - regular, lambda, nested - does not matter. Top level is basically the level of parameters/body
Will try make comment more clear.

---
In reply to: [135155951](https://github.com/dotnet/roslyn/pull/21334#discussion_r135155951) [](ancestors = 135155951)